src/stories/components/title-navigation-cu-about/title-navigation-cu-about.tsx

```diff
@@ -98,6 +98,14 @@ export const TitleNavigationCuAbout = ({ coreUnitAbout }: Props) => {
   const mips = getMipsStatus(orderMips[0] as CuMip);
   const mipStatus = (orderMips[0] as CuMip).mipStatus;
   const newDate = DateTime.fromFormat(mips || '', 'yyyy-MM-dd').toJSDate();
+
+  const formatCode = (code: string) => {
```

Comment:
I know that this function is relate only with the code of the core unit, but what do you thing if we move to other files like utils.ts, or something like that, to leave the component more clear?
Let me know what you think about it??

Reply:
I mean. It could be used on the future, although I'm not so sure about it. I'm still gonna move it there,